src/main/java/lotto/step3/domain/WinningNumbers.java

```diff
@@ -0,0 +1,64 @@
+package lotto.step3.domain;
+
+import lotto.step3.enums.Rank;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class WinningNumbers extends ValidateNumbers {
+    private final Set<Integer> winningNumbers;
+    private final int bonusNumber;
+    private static final int SECOND_POSITION = 5;
+
+    public Set<Integer> getWinningNumbers() {
+        return Set.copyOf(winningNumbers);
+    }
+
+    public WinningNumbers(Set<Integer> winningNumbers, int bonusNumber) {
+        this.winningNumbers = validateNumbers(winningNumbers);
+        this.bonusNumber = validateBonusNumber(bonusNumber, winningNumbers);
+    }
```

Comment:
```suggestion
public class WinningNumbers extends ValidateNumbers {
    private static final int SECOND_POSITION = 5;
    private final Set<Integer> winningNumbers;
    private final int bonusNumber;

    public WinningNumbers(Set<Integer> winningNumbers, int bonusNumber) {
        this.winningNumbers = validateNumbers(winningNumbers);
        this.bonusNumber = validateBonusNumber(bonusNumber, winningNumbers);
    }

    public Set<Integer> getWinningNumbers() {
        return Set.copyOf(winningNumbers);
    }
```
- 상수, 필드, 생성자, public 메소드, private 메소드 등 클래스 내에서 위치하는 순서들도 컨벤션이 있어요.

Reply:
아하!
감사합니다!